module/zfs/dbuf.c

```diff
@@ -1123,6 +1123,7 @@ static int
 dbuf_read_impl(dmu_buf_impl_t *db, zio_t *zio, uint32_t flags)
 {
 	dnode_t *dn;
+	arc_buf_t *dn_buf;
```

Comment:
nit: while you're here how about renaming this `abuf`, I think `dn_buf` is misleading and inconsistent with the rest of the code.